chemprop/cli/train.py

```diff
@@ -49,6 +51,16 @@
 
 logger = logging.getLogger(__name__)
 
+TASK_TYPES = [
+    "regression",
+    "regression-mve",
+    "regression-evidential",
+    "classification",
+    "classification-dirichlet",
+    "multiclass",
+    "multiclass-dirichlet",
+]
+
```

Comment:
It seems to me that we can replace this with `PredictorRegistry` using `from chemprop.nn.predictors import PredictorRegistry`

Reply:
The one difference between the two is that we haven't supported "spectral" yet, and it is in PredictorRegistry. See my other comment about raising a ValueError in `summarize`.